testtools/matchers/_impl.py

```diff
@@ -71,33 +122,14 @@ def __init__(self, description=None, details=None):
         self._details = details
 
     def describe(self):
-        """Describe the mismatch.
-
-        This should be either a human-readable string or castable to a string.
-        In particular, is should either be plain ascii or unicode on Python 2,
-        and care should be taken to escape control characters.
-        """
+        """Describe the mismatch."""
```

Comment:
I'd rather have no docstring here rather than an abreviated one, perhaps a decorator to consume the base class docstring?